.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: "ci"
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  workflow_dispatch:
+    
+jobs:
+  Spec:
```

Comment:
do we need to pass a ruby version matrix here? 

Something like this?
`  matrix:`
`      ruby_version:`
`       - '2.5'`
`       - '2.7'`
`    with:`
 `     ruby_version: ${{ matrix.ruby_version }}`

Reply:
ah great. Cheers